client/build.gradle

```diff
@@ -72,17 +72,21 @@ dependencies {
 
     implementation 'com.github.mrmike:ok2curl:0.6.0'
 
+    implementation 'com.facebook.stetho:stetho-okhttp3:1.5.1'
```

Comment:
I think we could make this be `debugImplementation` only, with [something like this](https://stackoverflow.com/a/31483962/4465208). Just to make sure we're not including it in release versions.